panel/tests/widgets/test_misc.py

```diff
@@ -78,10 +79,8 @@ def test_file_download_file():
 
 
 def test_file_download_callback():
-    file_download = FileDownload(callback=lambda: StringIO("data"), file="abc")
-
     with pytest.raises(ValueError):
-        file_download._clicks += 1
+        FileDownload(callback=lambda: StringIO("data"))
```

Comment:
Its moved up because I added the below.

![image](https://github.com/holoviz/panel/assets/42288570/b2b1ee81-b7f8-4309-8188-f9b80a600c0e)

I did it because I believe it is the right thing to do. Was it forgotten before?

Reply:
Why would calling `_update_filename` trigger a `ValueError`?

Reply:
Don't disagree calling it there is incorrect.